src/App.tsx

```diff
@@ -1,61 +1,38 @@
+import React, { useState } from 'react';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { Todo } from './components/types/Todo';
+import { getUserById } from './services/getUserById';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
-export const App = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <form action="/api/todos" method="POST">
-        <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
-        </div>
-
-        <div className="field">
-          <select data-cy="userSelect">
-            <option value="0" disabled>
-              Choose a user
-            </option>
-          </select>
-
-          <span className="error">Please choose a user</span>
-        </div>
+function getNewTodoId(todos: Todo[]) {
+  return Math.max(...todos.map(todo => todo.id)) + 1;
+}
```

Comment:
You can move it to a separate file as you did with getUserById

Reply:
> You can move it to a separate file as you did with getUserById

Good point